www/include/Administration/parameters/general/form.php

```diff
@@ -214,6 +215,34 @@
 $options3[] = $form->createElement('checkbox', 'yes', '&nbsp;', '');
 $form->addGroup($options3, 'enable_gmt', _("Enable Timezone management"), '&nbsp;&nbsp;');
 
+/*
+ * Keycloak
+ */
+$keycloak_enable[] = $form->createElement(
+    'checkbox',
+    'yes',
+    '&nbsp;',
+    '',
+    array("onchange" => "javascript:confirm("
+        ."'Are you sure you want to change this parameter ? Please read the help before.')"
```

Comment:
```suggestion
        . "'Are you sure you want to change this parameter ? Please read the help before.')"
```

Reply:
you should use a gettext message to be able to translate the message.
ie : you can use `<?= _("yourText"); ?>`